src/script/lua_api/l_mainmenu.cpp

```diff
@@ -3,6 +3,7 @@
 // Copyright (C) 2013 sapier
 
 #include "lua_api/l_mainmenu.h"
+#include "gui/touchscreeneditor.h"
```

Comment:
Please try to sort new includes a little bit so that it's easier to find duplicates, or relatives.